verification/verifier/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/veraison/services/verification/verifier
+
+go 1.18
```

Comment:
Question: Are we sticking with 1.17 or go 1.18?

Reply:
we want 1.18 for the generics on which the TTL cache depends

Reply:
Does the CI works with new go revision?

Reply:
I'd hope so, why?

Reply:
We had an issue in CI tests when we moved to 1.18 : Remember the conversation we had about this..

Reply:
not really, my memory is in terrible state :(